portfolio.css

```diff
@@ -2108,4 +1064,805 @@ input[type=email] {
+    border-radius: 8px;
+    background-image: url('./images/mobile_version/Card/Img_Plaholder.png') 30% 30% no-repeat;
+  }
+
+  .desktop-frameworks-list {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    padding: 0;
+    gap: 24px;
+    width: 337px;
+    height: 110px;
+  }
+
+  .desktop-framework-header {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 24px;
+    line-height: 32px;
+    text-align: center;
+    color: #344563;
+  }
+
+  .desktop-frame-tags {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    flex-wrap: wrap;
+    justify-content: center;
+    padding: 0;
+    gap: 8px;
+    width: 280px;
+    height: 88px;
+  }
+
+  .desktop-frame-tags > li {
+    padding: 12px;
+    list-style-type: none;
+    background: #fff;
+    border-radius: 8px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    color: #36b37e;
+  }
+
+  .desktop-frame-tags .frame-21 {
+    order: 2;
+  }
+
+  .desktop-frame-tags .frame-22 {
+    order: 0;
+  }
+
+  .desktop-frame-tags .frame-23 {
+    order: 1;
+  }
+
+  .desktop-skills-cards {
+    width: 320px;
+    height: 300px;
+    background: #ebf0ee;
+    border: 1px solid #c1c7d0;
+    border-radius: 8px;
+    background-image: url('./images/mobile_version/Card/Img_Plaholder.png') 30% 30% no-repeat;
+  }
+
+  .desktop-skills-list {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    padding: 0;
+    gap: 24px;
+    width: 337px;
+    height: 110px;
+  }
+
+  .desktop-skills-header {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 24px;
+    line-height: 32px;
+    text-align: center;
+    color: #344563;
+  }
+
+  .desktop-skills-frame {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    flex-wrap: wrap;
+    justify-content: center;
+    padding: 0;
+    gap: 8px;
+    width: 290px;
+    height: 88px;
+  }
+
+  .desktop-skills-frame > li {
+    padding: 12px;
+    list-style-type: none;
+    background: #fff;
+    border-radius: 8px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    color: #36b37e;
+  }
+
+  .desktop-skills-frame .skills-24 {
+    order: 0;
+  }
+
+  .desktop-skills-frame .skills-25 {
+    order: 1;
+  }
+
+  .desktop-skills-frame .skills-26 {
+    order: 2;
+  }
+
+  .desktop-skills-frame .skills-27 {
+    order: 3;
+  }
+
+  .desktop-skills-frame .skills-28 {
+    order: 0;
+  }
+
+  .desktop-form {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    justify-content: center;
+    padding: 80px 0;
+    gap: 24px;
+    width: 2063px;
```

Comment:
- [ ] Kindly note that _**Contact**_ section for the desktop view is currently not centered. This is attributed to the width you specified for the section ( width: 2063px; ). Kindly remove the width and the section will be centralized 👍🏽 .

![image](https://user-images.githubusercontent.com/63473584/187044592-a9f52b8d-8a56-41b0-ac46-95d5baa1cfc7.png)